### DISCUSSION TITLE
save metric output of val.py

### DISCUSSION THREAD

**Discussion by mar-du** [ultralytics/yolov5#8329] [Q&A]
Hi There,
I want to test a bunch of trained models and could not find a way to save the metrics (P, R, mAP05, mAP@.5, mAP@.5:.95) to a file instead of copy and paste them out of the terminal.
I tried to do it as I usually do with "> myFile.txt" after a val.py command in my bashscript but it didn´t work.
I assume I'm not the only one who would like to save the metrics, is there perhaps an implemented way I've overlooked?
Thanks!

**Comment by glenn-jocher** [comment]
@mar-du see https://github.com/ultralytics/yolov5/pull/8331 as an example of textfile logging. Does this work for your usecase?

**Comment by mar-du** [comment]
@glenn-jocher yes it works! thanks a lot!
